proto/internal/temporal/server/api/historyservice/v1/request_response.proto

```diff
@@ -1147,3 +1147,17 @@ message UpdateActivityOptionsResponse {
     // Activity options after an update
     temporal.api.activity.v1.ActivityOptions activity_options = 1;
 }
+
+// (-- api-linter: core::0134::request-mask-required=disabled
+// (-- api-linter: core::0134::request-resource-required=disabled
```

Comment:
nit: Do we need these here?

Reply:
yes. otherwise I'm getting linter errors.

Reply:
actually may be not...